barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_circuit_builder.cpp

```diff
@@ -1020,6 +1044,17 @@ template <typename Arithmetization> void UltraCircuitBuilder_<Arithmetization>::
     std::sort(list.variable_indices.begin(), list.variable_indices.end());
```

Comment:
This looks slow, did you run benchmarks?

Reply:
I measured at least +1s in WASM (measurements were kind of noisy, not sure the actual effect).